docs/Deploy/public-ip.md

```diff
@@ -1,8 +1,5 @@
 # Public IPs of devices
 
-> IMPORTANT: In order for this feature to work properly, devices must connect to Fleet via the public internet.
-> If the agent connects to Fleet via a private network then the "Public IP address" for such device will not be set.
-
```

Comment:
I'm actually removing this on my PR alongside removing the last sentence (in case that unblocks this PR).

Reply:
Ok I'll not change docs on my PR then!